docs/manuals/source/manually_added_config_directive_descriptions/dir-director-LogTimestampFormat.rst.inc

```diff
@@ -0,0 +1,5 @@
+This parameter needs to be a valid strftime format string. 
+See :command:`man 3 strftime` for the full list of available substitution
+variables.
+
+The default is :strong:`%d-%b %H:%M`.
```

Comment:
I think we should use the standard way here. I.e. just set the default in bareos itself and use the autogenerated files.

Reply:
```suggestion
```